website/docs/guides/provider_configuration.html.markdown

```diff
@@ -10,12 +10,50 @@ description: |-
 
 The New Relic Terraform Provider supports two methods of configuring the provider.
 
-1. [Using environment variables](#configuration-via-environment-variables) (recommended)
 1. [Using the `provider` block](#configuration-via-the-provider-block)
+2. [Using environment variables](#configuration-via-environment-variables) (recommended)
```

Comment:
You mention we recommend to use environment variables below. So it would move to to `1.` here.

Reply:
I honestly think the environment variables part created more toil for customers because it involves configuring the provider in multiple documents so I also removed "recommended". I think having the configuration docs start with everything in one file makes things easier to digest, especially for a new customer.